tox.ini

```diff
@@ -170,7 +190,6 @@ passenv = USERNAME
 extras =
     test
 deps =
-    {[plotmin]deps}
```

Comment:
Is there an inconvenience to remove this dependency? As far as I see, `testenv:pytest_mpl` is not used alone, it is reused in other environments where dependencies are already specified.

Reply:
yeah but if you wanted to run it alone locally, it would need those extra dependencies, no?

Reply:
at least I am getting a whole bunch of failure when trying `tox run -e pytest_mpl` on this PR locally